installer/templates/phx_web/templates/page/index.html.eex

```diff
@@ -1,5 +1,5 @@
 <section class="phx-hero">
-  <h1><%%= gettext "Welcome to %{name}!", name: "Phoenix" %></h1>
```

Comment:
I'd prefer to leave this since it's an example for new users to see. Can you update?

Reply:
Fixed on: https://github.com/phoenixframework/phoenix/pull/3387/commits/124cb48efa294f0fbf0b9832e65256458d7419c0